raiden/network/proxies/payment_channel.py

```diff
@@ -26,6 +27,10 @@ def __init__(
             token_network: TokenNetwork,
             channel_identifier: typing.ChannelID,
     ):
+
+        if 0 > channel_identifier > UINT64_MAX:
```

Comment:
`channel identifier` is a uint256 not a uint64. Check [here](https://github.com/raiden-network/raiden-contracts/blob/f5e106591c7738f70e14e4975b109fd0825d5816/raiden_contracts/contracts/TokenNetwork.sol#L32).

Also this check does not do what you think it does. Don't chain comparison operators like this without being sure what they are doing. This is equivalent to `if channel_identifier < 0 and channel_identifier > UINT64_MAX:` which is always False, even when you intended it to be true.